java/src/main/java/com/codurance/training/tasks/service/Impl/DeadlineServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package com.codurance.training.tasks.service.Impl;
+
+import com.codurance.training.tasks.Task;
+import com.codurance.training.tasks.service.DeadlineService;
+
+import java.io.PrintWriter;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class DeadlineServiceImpl  implements DeadlineService {
+    private Map<String, List<Task>> tasks = new LinkedHashMap<>();
```

Comment:
remove initialisation
add constructor parameter and initialise it
Logic also will fail

Reply:
removed initialisation and modified code